src/system_simulation/trace_visualization.rb

```diff
@@ -0,0 +1,126 @@
+# Simulate data samples of a system simulation by plotting a trace consisting of transparent circles into the scene.
+class TraceVisualization
+  def initialize
+    # Simulation data to visualize
+    @simulation_data = nil
+
+    # Group containing trace circles.
+    @group = Sketchup.active_model.active_entities.add_group
+
+    # List of trace circles.
+    @trace_points = []
+
+    # Visualization parameters
+    @color = Sketchup::Color.new(72,209,204)
+    @alpha = 0.4
+  end
+
+  def add_trace(node_ids, sparce_factor, data)
```

Comment:
I don't like the name sparce_factor, as it is not used as a factor of how many points we use, but the nth of wich points we take.

Reply:
Maybe sample_rate

Reply:
Done in 74d6b6a